2024Robot_Team3484/src/main/include/subsystems/IntakeSubsystem.h

```diff
@@ -0,0 +1,57 @@
+#ifndef INTAKE_H
+#define INTAKE_H
+
+
+#include <frc2/command/SubsystemBase.h>
+#include <rev/CANSparkMax.h>
+//#include <ctre/Phoenix.h>
+#include <frc/DigitalInput.h>
+#include <frc/trajectory/TrapezoidProfile.h>
+
+#include <units/angle.h>
+
+#include <Constants.h>
+
+class IntakeSubsystem : public frc2::SubsystemBase {
+    public:
+        IntakeSubsystem(
+            int PIVOT_MOTOR_CAN_ID, 
+            int DRIVE_MOTOR_CAN_ID, 
+            int PIECE_SENSOR_DI_CH, 
+            int ARM_SENSOR_DI_CH,
+            
+            double PID_P,
+            int PID_I,
+            int PID_D,
+            int PID_IZ_ZONE,
+            int PID_FF,
+            int PID_OUTPUTRANGE_MIN,
+            int PID_OUTPUTRANGE_MAX
```

Comment:
Verify the datatype the PID controller expects here. I believe they should be `double` not `int` datatypes here. You can also cut down the number of parameters declared here by using `SC_PIDConstants` (you just need to include `FRC3484_Lib/utils/SC_Datatypes.h`).